monorepo-migration/convert-to-hg.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
+
+set -e
+set -x
+
+CINNABAR=https://github.com/jcristau/git-cinnabar
+CINNABAR_BRANCH=detect-more-renames
```

Comment:
Nit: Let's add a code comment to remind future readers these value will eventually be changed.

Reply:
Well, I'm not sure they will...

Reply:
I'm open to adding a config to tweak rename/copy detection in git-cinnabar.

Reply:
Oh cool.  I'll look into that thenm thanks @glandium

Reply:
Added a comment for now.